lib/state/appState.js

```diff
@@ -54,12 +54,12 @@ appState.getInitial = function() {
     //   name: 'OneTouch UltraMini',
     //   key: 'onetouchmini',
     //   source: {type: 'device', driverId: 'OneTouchMini'}
+    // },
+    // {
+    //   name: 'Bayer Contour Next',
+    //   key: 'bayercontournext',
+    //   source: {type: 'device', driverId: 'BayerContourNext'}
     // }
-    {
```

Comment:
We haven't QA-ed it yet. It's not going to be tested until after the BtUTC release.